### DISCUSSION TITLE
Release frequency / process

### DISCUSSION THREAD

**Discussion by jahrome** [DefectDojo/django-DefectDojo#7649] [General]
Hi, 
I submitted a PR last week that was merged into bugfix. I was just wondering when a new release containing this fix will be issued or what the drivers for triggering a new release since it's blocking our deployment.
Thanks a lot and keep up the good work !

**Comment by mtesauro** [comment]
From a high level, there are monthly releases with bugfix releases as needed every week between 'proper' releases.

Using fictitious numbers, the basic way releases works is as follows:

Week 1: Create a new release e.g. 9.44.0
Week 2: Create a bugfix release e.g. 9.44.1
Week 3: Create a bugfix release e.g. 9.44.2
Week 4: Create a bugfix release e.g. 9.44.3

Week 5: Reset to Week 1 and start over with an new release.

Note: Depending on how things fall out on the calendar, there could be  a 4th bugfix release e.g. 9.44.4 some months.

HTH
